rules/cross-platform/privilege_escalation_sudoers_file_mod.toml

```diff
@@ -14,14 +14,14 @@ index = ["auditbeat-*", "logs-endpoint.events.*"]
 language = "kuery"
 license = "Elastic License"
 name = "Sudoers File Modification"
-risk_score = 21
+risk_score = 47
 rule_id = "931e25a5-0f5e-4ae0-ba0d-9e94eff7e3a4"
-severity = "low"
-tags = ["Elastic", "Host", "Linux", "Threat Detection", "Privilege Escalation"]
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "macOS", "Threat Detection", "Privilege Escalation"]
 type = "query"
 
 query = '''
-event.category:file and event.type:change and file.path:/etc/sudoers
+event.category :"file" and event.type:"change" and file.path:(/etc/sudoers* or /private/etc/sudoers*)
```

Comment:
```suggestion
event.category:file and event.type:change and file.path:(/etc/sudoers* or /private/etc/sudoers*)
```

Reply:
the quotes don't really matter but for consistency with the other KQL rules, we dont use them for those fields.